app/src/main/java/com/swent/suddenbump/model/meeting/Meeting.kt

```diff
@@ -0,0 +1,10 @@
+package com.swent.suddenbump.model.meeting
+
+import com.google.firebase.Timestamp
+
+data class Meeting(
+    val meetingId: String = "",
+    val location: String = "",
```

Comment:
For future PRs: would be good to have support for proper Location class. This would allow interaction with external navigation/directions apps.